src/commands/release/export.ts

```diff
@@ -0,0 +1,110 @@
+/**
+ * @license
+ * Copyright 2016-2024 Balena Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { commitOrIdArg } from '.';
+import { Flags } from '@oclif/core';
+import Command from '../../command';
+import * as cf from '../../utils/common-flags';
+import { getBalenaSdk, stripIndent } from '../../utils/lazy';
+import { create } from '@balena/release-bundle';
+import * as fs from 'fs/promises';
+import * as semver from 'balena-semver';
+import { ExpectedError } from '../../errors';
+
+export default class ReleaseExportCmd extends Command {
+	public static description = stripIndent`
+		Saves a release into a file.
+
+        Saving a release to a file allows you to import an exact 
+		copy of the original release into another application.
+
+		Only successful releases can be saved into a file.
+`;
+	public static examples = [
+		'$ balena release export a777f7345fe3d655c1c981aa642e5555 -o ../path/to/release.tar',
+		'$ balena release export 1234567 -o ../path/to/release.tar',
+		'$ balena release export myOrg/myFleet:1.2.3 -o ../path/to/release.tar',
```

Comment:
This looks very unintuitive 😕 🤔 @otaviojacobi what would you say is the correct/better way to ask the user for a semver which is bound to a specific fleet?

Reply:
@myarmolinsky , is it still unintuitive? I believe many users would be already used to how a docker tag format (`<repo|tag>:<version>`) looks like and we are just going with a similar convention.  I assume this concern is because we are referring to a fleet and not an image repository/tag.

Alternatively, we can go with the similar convention of `balena os download <device_type> --version <version>` if that provides consistency across our UI. cc @dfunckt just to let him know I am suggesting an alternative.

Reply:
@myarmolinsky @otaviojacobi , I changed the approach to use the flag `--version` instead similar to how he use the flag for OS downloads.  I also removed the example that uses the release ID since we don't recommend this approach anymore.